README.md

```diff
@@ -1 +1,8 @@
-# universal-application-tool
\ No newline at end of file
+# universal-application-tool
+
+
+To build the container that runs the app, type `docker build -t uat .`
+
+To run the application, run `docker run -it -p 9000:9000 -v ${PWD}/universal-application-tool-0.0.1:/code uat sbt ~run`
```

Comment:
let's put this in a script at `bin/run-dev` for convenience

Reply:
Done.